Library/TelemetryClient.ts

```diff
@@ -29,10 +29,10 @@ class TelemetryClient {
 
     /**
      * Constructs a new client of the client
-     * @param iKey the instrumentation key to use (read from environment variable if not specified)
+     * @param setupString the Connection String or Instrumentation Key to use (read from environment variable if not specified)
      */
-    constructor(iKey?: string) {
-        var config = new Config(iKey);
+    constructor(setupString?: string) {
```

Comment:
Do you have unit tests to make sure this still initializes correctly with either an Ikey or ConnString? Maybe I missed it but i didn't see any

Reply:
Yep these are in `Config.ts` tests.